unstructured/partition/xlsx.py

```diff
@@ -65,6 +62,9 @@ def partition_xlsx(
     for sheet_name, table in sheets.items():
         page_number += 1
         html_text = table.to_html(index=False, header=False, na_rep="")
+        html_string_parser = (
+            soupparser_fromstring if contains_emoji(html_text) else document_fromstring
+        )
```

Comment:
given that the test results are worse with `document_fromstring` in test_unstructured/partition/test_auto.py right now, i think it is just better to always use soupparser_fromstring.

Reply:
Sounds reasonable! After revisiting this, I also think it is better to always use `soupparser_fromstring`.